packages/lndr/format/index.ts

```diff
@@ -183,7 +182,7 @@ function convertToCommaDecimal(amount: string) : string {
 }
 
 export const convertCommaDecimalToPoint = (amount: string, currency: string) : string => {
-  if(!isCommaDecimal(currency)) {
```

Comment:
Typescript is throwing a new error here, because this function no longer uses the 'currency' property. It should be removed from this function's parameters and from the one place it is called in 'settlement/index.tsx'